SparkPost/2web2ui
# FE-785 - Billing Page Upgrades (Cypress)

### What Changed
 - What changes did you make to satisfy the AC?
     > Added cypress tests for account billing upgrade flows.

### How To Test
 - Run cypress locally and confirm it passes
 - Review the tests written and evaluate effectiveness. 
 - Review the tests and try to identify things to test that are not already.

### To Do
